cardwindow.cpp

```diff
@@ -1,16 +1,19 @@
 #include "cardwindow.h"
 #include "ui_cardwindow.h"
+#include "iostream"
```

Comment:
to chyba zostało po debugach
Przy okazji możesz poczytać o różnicy między "" a <> w inludach

Reply:
mozesz mergować